<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultado2</class>
 <widget class="QDialog" name="resultado2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fidelidad de la imagen</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Imagen original</string>
   </property>
   <widget class="QLabel" name="lblImagen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnCargar1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>120</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Imagen Alterada</string>
   </property>
   <widget class="QLabel" name="lblImagen2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtnCargar2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fidelidad de la imagen alterada:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFidelidad">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QProgressBar" name="pbarra">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnCalcular">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cantidad de elementos no afectados:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCantENA">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
